core/lib/refinery/plugin.rb

```diff
@@ -1,5 +1,16 @@
 module Refinery
   class Plugin
+    META = {
```

Comment:
> [...] especially when I have their position here as well (it will be in the next PR).
Just to be sure, there is already a config to set order for the menu : https://github.com/refinery/refinerycms/blob/master/core/lib/refinery/core/configuration.rb#L34

I think, we should keep it simple : 
-  define the icon in plugin registration
- let the user to decide how to organize their plugins, aka no feature for menu positions 1-50.